doc/source/Platforms.rst

```diff
@@ -456,6 +485,15 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
    spack concretize
    spack install [--verbose] [--fail-fast]
 
+.. note::
```

Comment:
I think this could be a general statement that could go in the quickstart section istead of the macOS specific instructions?

Reply:
Definitely good to have and to recommend to users and developers!

Reply:
I moved this to the Quickstart section.